packages/eslint-config/plugins/node-mjs.js

```diff
@@ -0,0 +1,14 @@
+// plugin:node/recommended: https://github.com/mysticatea/eslint-plugin-node/blob/master/lib/index.js
+module.exports = {
+  // considers all files as ES Modules.
+  extends: ['plugin:node/recommended-module'],
+  rules: {},
+  // some config file also use commonjs module
+  globals: {
```

Comment:
can `env: {node: true}` replace this?

Reply:
I use `env: { node: true}` in `index.js`(the root config), but can not override the globals which in `plugin:node/recommended-module`.

Reply:
Then you can add it here

Reply:
No, it does not work.
Seems `global` has high priority than `env`.So I need use `globals` here to override the config in `plugin:node/recommended-module`.

Reply:
A comment for `why not env here` would be nice.